pytorch_lightning/utilities/memory.py

```diff
@@ -75,5 +76,13 @@ def is_out_of_cpu_memory(exception):
 def garbage_collection_cuda():
     """Garbage collection Torch (CUDA) memory."""
     gc.collect()
-    if torch.cuda.is_available():
-        torch.cuda.empty_cache()
+    try:
+        if torch.cuda.is_available():
```

Comment:
This if can be moved outside of the try/catch, right?

Reply:
Yep.

Reply:
can you make the change? 😄